src/pages/ExamplePage.module.css

```diff
@@ -1,21 +1,33 @@
-.appShellAsideMobile {
-	top: calc(100% - 300px);
-	height: 300px;
-	max-height: 100%;
+.appShellMain {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    height: calc(100vh - 60px);
+    grid-template-areas:
+    "main main main"
```

Comment:
@snabela quite simple grid definition (3x3 area and fill it by predefined lock names)

Reply:
@LemToUp ah, this is a great solution!